typings/@chainlink/index.d.ts

```diff
@@ -1,14 +1,14 @@
 // Declare missing type definitions
 declare module '@chainlink/types' {
   export type AdapterRequestMeta = {
-    availableFunds: number
-    eligibleToSubmit: boolean
-    latestAnswer: number
-    oracleCount: number
-    paymentAmount: number
-    reportableRoundID: number
-    startedAt: number
-    timeout: number
+    availableFunds?: number
```

Comment:
I don't think all of this should be optional? Meta itself could be optional in the request, but if meta is set then all of this params should be set.

Reply:
It was throws errors from things like `iv-outlier-detection` adapter where meta gets used. 

I could go through and modify these, but might be better suited for a new 1 story point PR/issue as I didn't dig into Jonas' adapters to see how much needs to be changed.